### PR TITLE
Plot loop

### PR DESCRIPTION
**Pull request by Fil8**
plot loop is quick, all fucntions "source finder/ catalog from NVSS and pybdsm' work.

continuum image is not needed if catalog from NVSS or pybdsm are set to true

### CODE REVIEW

File: sharpener/sharp_modules/convert_units.py [hunk 1]
```diff
@@ -187,31 +187,24 @@ def coord_to_pix(imagename,ra,dec,verbose=False):
...
-				count_out +=1
-				if verbose == True:
-					print '# Source # '+str([i])+ ' lies outside the fov of the data cube #'
+				print '# Source # '+str([i])+ ' lies outside the fov of the data cube #'
 
 	print '# Total number of sources: \t'+str(len(ra))
-	print '# Sources below threshold: \t'+str(count_flag)
 	print '# Sources outside f.o.v.:\t'+str(count_out)
-	print '# Sources to analyze: \t\t'+str(len(ra)-count_flag-count_out)+'\n'
+	print '# Sources to analyze: \t\t'+str(len(ra)-count_flag-count_out)
```
Comment (Athanaseus) [inline sharpener/sharp_modules/convert_units.py]: might as well make the print statements py3 i.e.` print( '# Sources to analyze: \t\t'+str(len(ra)-count_flag-count_out))`

**Review by Athanaseus — APPROVED** [review]
Looks good to me.

File: sharpener/sharp_modules/convert_units.py (hunk 1 above)
Comment (Fil8) [reply sharpener/sharp_modules/convert_units.py]: yep, will do

**Review by Athanaseus — APPROVED** [review]

**Review by Fil8 — COMMENTED** [review]
looks good

**Pull request merged**